Documents/Pseudocode/Pseudocode_Final

```diff
@@ -58,13 +58,13 @@ Class Network
 		vector<matrix> sum_nabla_b;
 		vector<matrix> sum_nabla_w;
 		vector<int> test_data_indices;
-		vector<int> mini_batch_indices;		//only used in train(), why not limit its scope to that function?
-		int num_correct;			//only used in train(), why not limit its scope to that function?
-		string training_data_filename;		//consider truncating "filename". unnecessary and unwieldy
+		vector<int> mini_batch_indices;		//only used in train(), why not limit its scope to that function? //I agree [-Ami]
+		int num_correct;			//only used in train(), why not limit its scope to that function? //Yes [-Ami]
+		string training_data;		//consider truncating "filename". unnecessary and unwieldy
```

Comment:
I'm still sticking to appending _filename to the member. C++ auto-completes anyways, and the name says exactly what it is: not the data, not the file, but the *file's name*